server/controllers/find/coursesController.test.ts

```diff
@@ -106,10 +114,14 @@ describe('CoursesController', () => {
           expect(organisationService.getOrganisation).toHaveBeenCalledWith(token, organisationId),
         )
 
+        const organisationsWithoutUnfindableCourse = organisations.filter(
+          organisation => organisation.id !== 'NOTFOUND',
+        )
```

Comment:
I think we could also reduce some of the looping by creating this variable higher up and then creating an extra variable with the unfindable course?